src/components/DataSession/OperationWizard.vue

```diff
@@ -158,25 +171,58 @@ function selectImage(inputKey, imageIndex) {
       </v-row>
     </v-card-text>
     <v-slide-y-reverse-transition hide-on-leave>
-      <v-card-text v-show="page == 'configure'" class="wizard-card">
-        <div v-for="(inputDescription, inputKey) in selectedOperationInputs" :key="inputKey"
-          class="operation-input-wrapper">
-          <v-text-field v-if="inputDescription.type !== 'file'" v-model="selectedOperationInput[inputKey]"
-            :label="inputDescription.name" :type="inputDescription.type" class="operation-input" />
-          <div v-else-if="inputDescription.type == 'file'" class="images-container">
-            <image-grid :images="images" :column-span="calculateColumnSpan(images.length, imagesPerRow)"
-              class="wizard-images" :allow-selection="true" @select-image="selectImage(inputKey, $event)" />
+      <v-card-text
+        v-show="page == 'configure'"
+        class="wizard-card"
+      >
+        <div
+          v-for="(inputDescription, inputKey) in selectedOperationInputs"
+          :key="inputKey"
+          class="operation-input-wrapper"
+        >
+          <v-text-field
+            v-if="inputDescription.type !== 'file'"
+            v-model="selectedOperationInput[inputKey]"
+            :label="inputDescription.name"
+            :type="inputDescription.type"
+            class="operation-input"
+          />
+          <div
+            v-else-if="inputDescription.type == 'file'"
+            class="images-container"
+          >
+            <div
+              v-if="inputDescription.name == 'Input Files'"
+              class="input-images"
+            >
+              Input Files
```

Comment:
No no I think you misunderstood. I want the text to be `{{ inputDescription.name }}` instead of "Input Files", that way it will show as "Input Files" in this case (where the name is Input Files), but for future operations, it will show whatever their input is called, without having to add a new case to the frontend. Does that make sense?

Reply:
yep yep, apologies